app/src/atoms/SoftwareKeyboard/AlphanumericKeyboard/index.tsx

```diff
@@ -2,18 +2,23 @@ import * as React from 'react'
 import Keyboard from 'react-simple-keyboard'
 import { alphanumericKeyboardLayout, customDisplay } from '../constants'
 
+import '../index.css'
+import './index.css'
+
+// Note (kk:04/05/2024) add debug to make debugging easy
 interface AlphanumericKeyboardProps {
   onChange: (input: string) => void
   keyboardRef: React.MutableRefObject<any>
+  debug?: boolean
 }
 
 export function AlphanumericKeyboard({
   onChange,
   keyboardRef,
+  debug = false,
```

Comment:
```suggestion
  debug = false, // If true, <ENTER> will input a \n
```

I think it might be worth leaving a comment here, since what this actually does isn't entirely intuitive (not your fault at all - a library issue).